smallLibraries/Google.Cloud.Templates/content/gcpmvc/CSharp/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,96 @@
+            <img src="~/images/aspnet-banner.svg" alt="ASP.NET" class="img-responsive" />
+            <div class="carousel-caption" role="option">
+                <p>
+                    Learn how to build ASP.NET apps that can run anywhere.
+                    <a class="btn btn-default" href="http://go.microsoft.com/fwlink/?LinkID=525028&clcid=0x409">
+                        Learn More
+                    </a>
+                </p>
+            </div>
+        </div>
+        <div class="carousel-item bg-dark">
+            <img src="~/images/google-cloud-banner.svg" class="img-responsive" />
+            <div class="carousel-caption" role="option">
+                <p>
+                    Develop in Visual Studio and deploy to Google Cloud Platorm with
+                    <a class="btn btn-default" href="https://marketplace.visualstudio.com/items?itemName=GoogleCloudTools.GoogleCloudPlatformExtensionforVisualStudio">
+                        Cloud Tools for Visual Studio
+                    </a>
+                </p>
+            </div>
+        </div>
+        <div class="carousel-item">
+            <img src="~/images/packaging-banner.svg" alt="Package Management" class="img-responsive" />
+            <div class="carousel-caption" role="option">
+                <p>
+                    Bring in libraries from NuGet, Bower, and npm, and automate tasks using Grunt or Gulp.
+                    <a class="btn btn-default" href="http://go.microsoft.com/fwlink/?LinkID=525029&clcid=0x409">
+                        Learn More
+                    </a>
+                </p>
+            </div>
+        </div>
+    </div>
+    <a class="carousel-control-prev" href="#myCarousel" role="button" data-slide="prev">
+        <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+        <span class="sr-only">Previous</span>
+    </a>
+    <a class="carousel-control-next" href="#myCarousel" role="button" data-slide="next">
+        <span class="carousel-control-next-icon" aria-hidden="true"></span>
+        <span class="sr-only">Next</span>
+    </a>
+</div>
+
+<div class="row mt-4">
+    <div class="col-md-3">
+        <h2>Application uses</h2>
+        <ul>
+            <li>Sample pages using ASP.NET Core MVC</li>
+            <li><a href="https://docs.npmjs.com/getting-started/what-is-npm">npm</a> for managing client-side libraries</li>
+            <li>Theming using <a href="http://go.microsoft.com/fwlink/?LinkID=398939">Bootstrap</a></li>
+        </ul>
+    </div>
+    <div class="col-md-3">
+        <h2>How to</h2>
+        <ul>
+            <li><a href="http://go.microsoft.com/fwlink/?LinkID=398600">Add a Controller and View</a></li>
+            <li><a href="http://go.microsoft.com/fwlink/?LinkID=699562">Add an appsetting in config and access it in app.</a></li>
+            <li><a href="http://go.microsoft.com/fwlink/?LinkId=699315">Manage User Secrets using Secret Manager.</a></li>
```

Comment:
We should replace these with the links to "how-to" guides on .NET on GCP.
[Setting up a .NET Development Environment](https://cloud.google.com/dotnet/docs/setup)
[Deploying apps to AppEngine Flex](https://cloud.google.com/appengine/docs/flexible/dotnet/quickstart)
[Writing and Responding to Pub/Sub messages](https://cloud.google.com/appengine/docs/flexible/dotnet/writing-and-responding-to-pub-sub-messages)

We should also provide links for:

1) [How to view logs](https://cloud.google.com/logging/docs/view/overview)
2) [Build a fully featured app on AppEngine](https://cloud.google.com/dotnet/docs/getting-started/tutorial-app)